web/src/billing/billing.ts

```diff
@@ -393,5 +390,13 @@ export function initialize(): void {
 }
 
 $(() => {
+    // Retrieve and display the success message.
+    const success_message = sessionStorage.getItem("success_message");
+    if (typeof success_message === "string") {
+        $("#billing-success-message-top").text(success_message);
+        $("#billing-success-message-top").show();
+        sessionStorage.removeItem("success_message");
+    }
```

Comment:
@timabbott I think this approach would work well. I didn't want to use `localstorage` to avoid user getting a success message on opening the billing page after a month in case `localstorage` was not cleared.